src/test/java/com/icthh/xm/ms/otp/web/rest/errors/ExceptionTranslatorIntTest.java

```diff
@@ -6,6 +6,7 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
```

Comment:
very strange changes, only one import? maybe it unused?

Reply:
done